It does seem like changing this only in memtailor fixes the issue, but technically mathic and mathicgb also have the same issue (if they were to be built independently).

This is the error btw:
```
-- Could NOT find GTest (missing: GTEST_LIBRARY GTEST_INCLUDE_DIR GTEST_MAIN_LIBRARY)
CMake Error at BUILD/build/_deps/googletest-src/CMakeLists.txt:4 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -46,7 +46,7 @@ if(BUILD_TESTING)
     include(FetchContent)
     FetchContent_Declare(googletest
       GIT_REPOSITORY https://github.com/google/googletest.git
-      GIT_TAG        release-1.16.0
+      GIT_TAG        v1.16.0
       )
     FetchContent_MakeAvailable(googletest)
 

